### ISSUE TITLE
What is the performance of this optimizer with small learning rate?

### ISSUE THREAD

**Issue by CoinCheung** [deepmind/lamb#9]
Hi,

I know the optimizer of lars/lamb is proposed in the background of large batchsize. What if the batch size is small such as training imagenet with batchsize=256 ? Will it be ok to use this optimizer in these occasions ?

**Comment by melisgl** [comment]
This repo is for the language modelling benchmarks code. It's unrelated to the lamb optimizer :-).
